app/src/main/res/layout-land/walkthrough_final_fragment.xml

```diff
@@ -43,12 +43,12 @@
         android:id="@+id/walkthrough_final_no_button"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="@dimen/margin_8"
+        android:layout_marginEnd="@dimen/space_8dp"
         android:layout_weight="1"
         android:background="?attr/selectableItemBackground"
         android:onClick="@{(v) -> presenter.goBack()}"
         app:cardCornerRadius="4dp"
-        app:contentPadding="@dimen/padding_24dp">
+        app:contentPadding="@dimen/space_24dpdp">
```

Comment:
it should be `space_24dp` not `dpdp`. 

here and all places.

Reply:
done